lib/sqlalchemy/dialects/postgresql/asyncpg.py

```diff
@@ -1060,10 +1086,24 @@ def do_terminate(self, dbapi_connection) -> None:
 
     def create_connect_args(self, url):
         opts = url.translate_connect_args(username="user")
+        multihosts, multiports = self._split_multihost_from_url(url)
 
         opts.update(url.query)
+
+        if multihosts:
+            if not all(multihosts):
```

Comment:
**mike bayer** (zzzeek) wrote:

is there any way that None could be present in hosts here?   there should be a test if so

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4755